playback/src/config.rs

```diff
@@ -139,6 +139,8 @@ pub struct PlayerConfig {
     // pass function pointers so they can be lazily instantiated *after* spawning a thread
     // (thereby circumventing Send bounds that they might not satisfy)
     pub ditherer: Option<DithererBuilder>,
+    // setting this will enable periodically sending events informing about playback position
```

Comment:
Please add another `/` so that this becomes a doc-comment, rather then a simple comment. And maybe also reference the enum variant here.